src/snark_intf.ml

```diff
@@ -289,6 +290,27 @@ module type Basic = sig
 
     val field : (Field.Var.t, field) t
 
+    val snarkless : 'a -> ('a, 'a) t
```

Comment:
Should we maybe put these into an "internal" module or something?

Reply:
Done, and added a comment to recommend avoiding them.